TSQL_Parser/TSQL_Parser/Clauses/TSQLWhenClause.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TSQL.Clauses
+{
+	public class TSQLWhenClause : TSQLClause
```

Comment:
We may have to think on this one. I can probably agree that it's a clause, but the same word is also used as a clause in a CASE.

Reply:
I think I've decided that when we start using this class also for WHEN inside a CASE, if it does not support features well for both cases, we can create subclasses that inherit TSQLWhenClause, e.g. TSQLMergeWhenClause and TSQLCaseWhenClause, without breaking backwards compatibility, so this class as it stands should be good for now.